components/authz-service/storage/v2/postgres/postgres.go

```diff
@@ -1502,6 +1509,70 @@ func (p *pg) CreateProject(ctx context.Context, project *v2.Project) (*v2.Projec
 	return project, nil
 }
 
+func (p *pg) addSupportPolicies(ctx context.Context, project *v2.Project, q Querier) error {
+	policyParams := []struct {
+		id   string
+		name string
+		role string
+	}{
+		{
+			id:   fmt.Sprintf("%s-%s", project.ID, "project-owners"),
+			name: fmt.Sprintf("%s %s", project.Name, "Project Owners"),
+			role: constants_v2.ProjectOwnerRoleID,
+		},
+		{
+			id:   fmt.Sprintf("%s-%s", project.ID, "project-editors"),
+			name: fmt.Sprintf("%s %s", project.Name, "Project Editors"),
+			role: constants_v2.EditorRoleID,
+		},
+		{
+			id:   fmt.Sprintf("%s-%s", project.ID, "project-viewers"),
+			name: fmt.Sprintf("%s %s", project.Name, "Project Viewers"),
+			role: constants_v2.ViewerRoleID,
+		},
+	}
+
+	for _, param := range policyParams {
+		pol := generatePolicy(project.ID, param.id, param.name, param.role)
+
+		if err := p.insertPolicyWithQuerier(ctx, &pol, q); err != nil {
+			return err
+		}
+
+		if err := p.associatePolicyWithProjects(ctx, pol.ID, pol.Projects, q); err != nil {
+			return err
+		}
+
+		if err := p.insertPolicyStatementsWithQuerier(ctx, pol.ID, pol.Statements, q); err != nil {
+			return err
+		}
+	}
+
+	if err := p.notifyPolicyChange(ctx, q); err != nil {
```

Comment:
Economy of scale: just do the requisite notification once for all 3 policies.

Reply:
Are these policies even worth the notification? seems like they're only meaningful if they have members. 🤔 That is perhaps a bigger question than this PR, though.